### PR TITLE
Add recipe for tox-current-env

### PR DESCRIPTION
**Pull request by millsks**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by millsks** [comment]
@conda-forge/staged-recipes, ready for review!

File: recipes/tox-current-env/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tox-current-env-{{ version }}.tar.gz
+  sha256: 5b05d7a169e323ca73c8bacd90cfcfe38e2672b705f944f94d1d6c0f56f31c51
+
+build:
+  skip: true  # [py2k]
```
Comment (ocefpaf) [inline recipes/tox-current-env/meta.yaml:13]:
```suggestion
  noarch: python
```

File: recipes/tox-current-env/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
+    - tox >=3.24
+    - importlib-metadata  # [py<38]
```
Comment (ocefpaf) [inline recipes/tox-current-env/meta.yaml:26]: We are on the brink of  dropping py37.

```suggestion
```

File: recipes/tox-current-env/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/tox-current-env/meta.yaml:24]:
```suggestion
    - python >=3.8
```

**Review by ocefpaf — COMMENTED** [review]
Making it noarch makes it easier to build and the only loss here is the py37 version that will be dropped in a few days.

**Comment by millsks** [comment]
Made the requested changes.  Thanks for the heads up about python 3.7.  I let the others know too.

**Comment by millsks** [comment]
@conda-forge/staged-recipes, ready for review!

**Comment by millsks** [comment]
@conda-forge/staged-recipes, ready for review!

**Pull request merged**